### ISSUE TITLE
GM cannot modify group members

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#232]
GM cannot modify group members

`Game Master (GM) says:You are not the owner of this group. You cannot modify its members.`

but you can set any group to be your active group
